html-aam/index.html

```diff
@@ -6420,7 +6497,46 @@ <h4 id="el-select-combobox">`select`<span class="el-context"> (with NO `multiple
             <!-- <th><a href="#accessible-name-and-description-computation">Naming Algorithm</a></th> -->
             <tr>
               <th>Comments</th>
-              <td></td>
+              <td>
+                <p>To calculate the accessible value of a `select` element:</p>
+                <ol>
+                  <li>If `aria-valuetext` is specified on the `select` element, use the attribute's value, including the empty string.</li>
+                  <li>Else if there is a `button` part, calculate the accessible value from a non-empty string <a data-cite="accname-1.2/#mapping_additional_nd_te">text equivalent computation</a> of the subtree of that element.</li>
```

Comment:
Is there anything in the HTML spec that prevents the author from putting something other than the value in the `button` part? For example, what if the author puts a label in the `button` part? It would be very weird to expose a label as the value. Obviously, we can't stop authors from doing this, but I wonder if they should at least be warned against it, though that probably belongs in the HTML spec itself.